res/xml/dvorak.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard>
+	<row>
+		<key key0="backspace" key1="esc" key2="`" key3="~" key4="delete" width="1.5" />
+		<key key0="shift" width="1.5" />
+		<key key0="p" key1="accent_ring" />
```

Comment:
My finger are less precise on the backspace key. Also it's useful to have an empty corner to be able to modulate the key repeat.
What do you think of moving these to the `p` ?

```suggestion
		<key key0="backspace" key1="esc" key4="delete" width="1.5" />
		<key key0="shift" width="1.5" />
		<key key0="p" key1="accent_ring" key2="`" key3="~" />
```

Reply:
Also, what about moving the backspace to the end of the top row ?

Reply:
I think having the accents on the P key is a great idea.

I'm less sure about moving the backspace, since it'd throw off the alignment of the other keys, but it might be fine? I've been using the layout contiuously since making the PR, and while I've settled into it, it does still feel a little odd. I haven't really struck upon a solution that appeals to me, but it's worth trying it with the backspace in a more normal position.

Reply:
> I would like to merge this soon, before the release. How is your experiment going ? Happy to merge once you mark the PR as ready.
> 
> There have been many changes on `master` so I pushed a merge commit.

Thanks for merging the changes.

I've been testing it since I made the PR, and have had a few issues, but I'm still mulling over what the solution might look like.